evaluate_model.py

```diff
@@ -0,0 +1,57 @@
+
+from pipeline import load_pipeline, load_recordings, get_epochs
+from sklearn.metrics import confusion_matrix
+import numpy as np
+
+
+def main():
+    model_subject_name = 'Haggai2'
+    data_subject_name = 'Haggai2'
+
+    # model
+    pipeline = load_pipeline(model_subject_name)
+
+    # data
+    raw, params = load_recordings(data_subject_name)
+    epochs, labels = get_epochs(raw, params["trial_duration"])
+    epochs = epochs.get_data()
```

Comment:
Epochs no longer returns an `mne.Epochs` object but the data directly so you can remove this line